updatecli/values.yaml

```
@@ -9,5 +9,5 @@ github:
   repository: "docker-packaging"
 
 jdk:
-  majorVersion: 17
-  agentMajorVersion: 17
+  majorVersion: 21
```

```suggestion
  majorVersion: 17
```

as we don't want to build Jenkins (yet?) with JDK21